Doc/c-api/object.rst

```diff
@@ -353,6 +353,15 @@ Object Protocol
    *NULL* on failure.
 
 
+.. c:function:: PyObject* PyObject_CallMethodObjNoArgs(PyObject *obj, PyObject *name)
```

Comment:
Please note that `ObjArgs` means "called with a variable number of `PyObject*` arguments".
This suffix is added because `PyObject_CallMethod` uses `Py_BuildValue` style arguments.

On the other hand, this API doesn't accept any arguments.  `CallMethodNoArgs` makes more sense.

Reply:
OK, I changed the name.